cmake/GinkgoConfig.cmake.in

```diff
@@ -91,6 +91,8 @@ set(GINKGO_HAVE_HWLOC @GINKGO_HAVE_HWLOC@)
 
 set(GINKGO_HAVE_ROCTX @GINKGO_HAVE_ROCTX@)
 
+set(GINKGO_HAVE_METIS @GINKGO_HAVE_METIS@)
```

Comment:
`GINKGO_HAVE_METIS` is already set below on line 140, but as a string (!?!)

Reply:
yes right, I thought we had grouped them correctly. I'll investigate once I can reproduce it

Reply:
That might have caused #1308 because [cmake documentation](https://cmake.org/cmake/help/latest/command/if.html) says:

> if(<string>)
> A quoted string always evaluates to false unless:
> 
> The string's value is one of the true constants, or
> 
> Policy CMP0054 is not set to NEW and the string's value happens to be a variable name that is affected by CMP0054's behavior.